flink-libraries/flink-table/src/test/scala/org/apache/flink/table/api/batch/table/validation/AggregateValidationTest.scala

```diff
@@ -208,4 +208,58 @@ class AggregateValidationTest extends TableTestBase {
     // must fail. UDAGG does not accept String type
     .select("myWeightedAvg(c, a)")
   }
+
```

Comment:
Add a test to validate aggregate with two aggregate functions?

Reply:
Done